extensions/amp-ad-network-adsense-impl/0.1/responsive-state.js

```diff
@@ -86,10 +86,7 @@ export class ResponsiveState {
    *  corresponds to a responsive ad, otherwise null.
    */
   static createIfResponsive(element) {
-    if (
-      !this.isContainerWidth_ &&
```

Comment:
While this PR doesn't change behavior, do we know what the original intent was?

Reply:
I think this should be changed to an instance method, and the intent is to check `this.isContainerWidth_` on the instance.

Reply:
My take is the intent is not clear.

I read this as a constructor replacement either returning null or a new class instance based on some conditions. One of these conditions is not possible.

Reply:
Additionally, refactoring this to no longer be static is significantly altering it and callers usage.

When @Jiaming-X has a chance to review, it would be good to learn what they prefer.

Reply:
Lastly, there is no test covering this usage. 

If there is an intent for this code to branch the logic within the `static` method, lets also add a test covering it.

Reply:
+1, intent isn't clear here. I'm fine